src/Sylius/Bundle/CoreBundle/Provider/AvailableLocalesProvider.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Provider;
+
+use Sylius\Component\Resource\Provider\AvailableLocalesProviderInterface;
+use Sylius\Component\Locale\Model\LocaleInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.org>
+ */
+class AvailableLocalesProvider implements AvailableLocalesProviderInterface
```

Comment:
Shouldn't be ``AvailableLocaleProvider`` (like [``LocaleProvider``](https://github.com/tuka217/Sylius/blob/sylius-translations/src/Sylius/Component/Resource/Provider/LocaleProvider.php)?)